README.md

```
@@ -300,6 +299,27 @@ Ripen は「1対1」ではなく「N対1」の接続を前提とした**ナレ
 
 **Ripenは `Streamable HTTP Hub` として動作します。** HTTPサーバーとして常駐し、複数のIDE・複数のメンバーが同時に読み書きできます。
 
+> **Docker配布の取りやめとマルチOS対応について**
+> 企業や商用環境での利用時に懸念されるDocker Desktop等のライセンス問題を考慮し、Dockerによるコンテナ配布方針は取りやめました。
+> .exeオンリーにした理由については、このRipenはWindows環境が一台あったら、その他のAIエージェントはHTTP通信をするだけであるので、ほとんどすでにマルチOS対応は完了しているも同然であると判断して、ライセンスの問題があるdockerでの配布方針を取りやめました。
+> チーム内にWindows環境が1台稼働していれば、MacやLinuxなど他のOSを使うメンバーのAIエージェントはHTTP通信（SSE）で接続するだけで利用可能です。これにより、配布形態をライセンス問題のないWindows `.exe` およびPythonソース起動に一本化しています。
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)